src/main/resources/templates/fragment/sidebar.html

```
@@ -0,0 +1,66 @@
+<!-- 사이드바 -->
+<aside th:fragment="copy" class="fixed top-0 left-0 h-full text-black flex flex-col justify-start py-6 w-56 bg-gray-900 border-r border-gray-800 shadow-lg" style="padding-top: 100px;">
```

HTMLHint (1.5.0)

[error] Doctype must be declared before any non-comment content.

(doctype-first)